README.md

```diff
@@ -36,6 +36,7 @@
 > There are two contributers for this project and we both eqully contributed
 > Here is screenshoot of the kanban board
 >
+
```

Comment:
# Frontend review

- [ ]  In `App.js`, please, kindly refactor this main `App` functional component to ES6 arrow function, so your code complies with the latest standards.

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/8c1643f4-58f5-4d6b-ac2c-b991f07e7198)


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards, and your project will be ready to prime time 💪